src/main/java/it/gov/pagopa/pu/debtpositions/service/InstallmentServiceImpl.java

```diff
@@ -73,4 +77,52 @@ public String updateInstallmentNotificationDate(UpdateInstallmentNotificationDat
     return debtPositionUpdateInstallmentService.updateInstallment(debtPositionDTO, updatedInstallments, wfExecutionParameters, accessToken, operatorExternalUserId).getRight();
   }
 
+  @Override
+  public InstallmentDTO updateInstallmentNotificationFee(Long organizationId, String nav, List<DebtPositionOrigin> debtPositionOrigin, long notificationFeeCents) {
+    List<InstallmentDTO> installments = installmentPIIRepository.getByOrganizationIdAndNav(organizationId, nav, debtPositionOrigin).stream()
+      .filter(installment ->
+        installment.getStatus().equals(InstallmentStatus.UNPAID) ||
+        installment.getStatus().equals(InstallmentStatus.EXPIRED) ||
+        installment.getStatus().equals(InstallmentStatus.UNPAYABLE))
+      .map(installmentMapper::mapToDto)
+      .toList();
+
+    if(installments.isEmpty())
+      throw new NotFoundException("The installment with NAV: "+nav+" was not found");
+    if(installments.size() > 1)
+      throw new ConflictErrorException("Found more than one installment processable with NAV: "+nav);
+
+    InstallmentDTO installment = calculateNewAmount(installments.getFirst(), notificationFeeCents);
+    installmentPIIRepository.save(installmentMapper.mapToModel(installment));
+
+    InstallmentUtils.setStatus(installment, installment.getStatus());
```

Comment:
don't do this, instead call it.gov.pagopa.pu.debtpositions.service.update.DebtPositionUpdateInstallmentService#updateInstallment (as done on updateInstallmentNotificationDate)
this way, it will validate the changes, align amounts, set status and align it and then invoke the sync WF